custom-vu/src/main/kotlin/jces1209/vu/page/customizecolumns/DcCustomizeColumns.kt

```diff
@@ -0,0 +1,38 @@
+package jces1209.vu.page.customizecolumns
+
+import com.atlassian.performance.tools.jiraactions.api.page.wait
+import org.openqa.selenium.By
+import org.openqa.selenium.WebDriver
+import org.openqa.selenium.WebElement
+import org.openqa.selenium.support.ui.ExpectedConditions
+import java.time.Duration
+
+class DcCustomizeColumns(
```

Comment:
DcCustomizeColumns and CloudCustomizeColumns have the same implementations  
Please move use one class

Reply:
for different customers might be differences, i'd leave as is.

Reply:
anyway it's necessary to avoid code duplication - then you could move the general implementation in the parent class (CloudEditor: interface -> abstract class)

Reply:
done